### ISSUE TITLE
Input v-model not working in vue project

### ISSUE THREAD

**Issue by merveyapnaz** [Trendyol/baklava#218]
**Describe the bug**
When we try to use the v-model in the input in the Vue project, we can't get and change the value.

**To Reproduce**

1. Add bl-input to page
`<bl-input type="text" v-model="value"></bl-input>`
2. Change value in input
3. Try get new value
4.  Try set value with code

**Expected behavior**
Expecting get changed value and set input value

**Desktop:**
 - OS: MacOS 12.5.1
 - Browser: Chrome
 - Version 2.0.0

**Comment by muratcorlu** [comment]
Thanks for reporting. I found the issue, working on it...

**Comment by muratcorlu** [comment]
@merveyapnaz A fix is just released with v2.0.0-beta.24 version. Can you try with that version? Please be sure that you import correct version like:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@trendyol/baklava@2.0.0-beta.24/dist/themes/default.css" />
<script type="module" src="https://cdn.jsdelivr.net/npm/@trendyol/baklava@2.0.0-beta.24/dist/baklava.js"></script>
```
